### PR TITLE
Added helplinks to all the elements and a missing tooltip text

### PR DESCRIPTION
**Pull request by Ananthavijay**
Fixes https://github.com/CircuitVerse/CircuitVerse/issues/1317

#### Describe the changes you have made in this PR -
- There are two buttons: 
1. CircuitVerse Tip - when hovered, the tooltip text of the element will be displayed.
2. CircuitVerse Help - when clicked, it redirects the user to the element's document page. If the element does not have documentation then this button will not be displayed.
- A tooltip text was missing for Random, added it.
#### Before:
Screenshot (552)

#### After:
Screenshot (557)

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by Ananthavijay** [comment]
@satu0king @nitin10s Please review!

File: public/js/UX.js [hunk 1]
```diff
@@ -216,19 +216,16 @@ function showProperties(obj) {
...
         });
-        //redirects to obj's specific help page
-        $('#toolTipButton').click(function () {
-            var helplink = obj && (obj.helplink);
+    }
+
+    var helplink = obj && (obj.helplink);
+    if (helplink) {
+       $('#moduleProperty-inner').append('<p><button id="HelpButton" class="btn btn-primary btn-xs" type="button" >CircuitVerse Help &#9432</button></p>');
+       $('#HelpButton').click(function () {
```
[the remark this answers is not in the document]
Comment (Ananthavijay) [reply public/js/UX.js]: Done

File: public/js/UX.js [hunk 2]
```diff
@@ -216,19 +216,16 @@ function showProperties(obj) {
...
             $("#Help").removeClass("show");
         });
-        //redirects to obj's specific help page
-        $('#toolTipButton').click(function () {
-            var helplink = obj && (obj.helplink);
+    }
+
+    var helplink = obj && (obj.helplink);
+    if (helplink) {
+       $('#moduleProperty-inner').append('<p><button id="HelpButton" class="btn btn-primary btn-xs" type="button" >CircuitVerse Help &#9432</button></p>');
```
[the remark this answers is not in the document]
Comment (Ananthavijay) [reply public/js/UX.js]: Done

**Comment by aayush-05** [comment]
The tooltip for CircuitVerse Tip is also visible when the elements are hovered on the sidebar, so an explicit button for the same is redundant.

**Comment by Ananthavijay** [comment]
I agree but that button was already implemented so that's why I didn't remove it.

**Comment by aayush-05** [comment]
Yes, let us wait for the confirmation by @satu0king

**Comment by Ananthavijay** [comment]
Removed the tooltip button and updated the screenshots.

**Comment by Ananthavijay** [comment]
@nitin10s For the test bench elements, can I add the same link? ( https://docs.circuitverse.org/#/testbench )

**Comment by nitin10s** [comment]
Yes

**Comment by Ananthavijay** [comment]
Done.
Elements which doesn't have documentation are: 
- RGBLedMatrix
- ForceGate
- Clock
- ControlledInverter

**Pull request merged**